rsa/src/sign.rs

```diff
@@ -0,0 +1,15 @@
+//! RSA signing key.
+
+use crate::Signature;
+use digest::DynDigest;
+use signature::{RandomizedSigner, Signer};
+
+/// RSA Signing key using PKCS1v15 padding
+pub trait PKCS1v15SigningKey: Signer<Signature> {}
+/// RSA Verifying key using PKCS1v15 padding
+pub trait PKCS1v15BlindedSigningKey: RandomizedSigner<Signature> {}
+
+/// RSA Signing key using PSS padding
+pub trait PSSSigningKey<D: DynDigest>: RandomizedSigner<Signature> {}
+/// RSA Verifying key using PSS padding
+pub trait PSSBlindedSigningKey<D: DynDigest>: RandomizedSigner<Signature> {}
```

Comment:
I'd suggest having separate `pkcs1v15::Signature` and `pss::Signature` types. Then you don't need separate traits like this. Instead they're just `Signer<pkcs1v15::Signature>` versus `Signer<pss::Signature>`.

The compiler can even infer which impl to use from the return type.